foreign_cc/private/framework/platform.bzl

```diff
@@ -1,8 +1,40 @@
 """A helper module containing tools for detecting platform information"""
 
+SUPPORTED_CPU = [
+    "aarch64",
+    "s390x",
+    "x86_64",
+]
```

Comment:
Ideally we'd mirror whats in platforms: https://github.com/bazelbuild/platforms/blob/main/cpu/BUILD but not sure what a good way of automating the extraction of those values is, especially as these are used to create Labels in the platforms repo..

Reply:
Yeah, I took these from platforms. Originally, I had every CPU in platforms in this list, and then I realized I wasn't actually sure how to map some of the cpus to gnu-triples or to cmake-system-names; I also didn't have a way to test them. My gut said it was potentially more confusing to get it wrong than to skip it until someone who could test came along and needed it, but I'm happy to restore the original list if you'd like.

One thing I also deliberated doing (even though it's slightly disapproved of in the bazel docs) was issue a `print()` warning every time the unknown case was detected, with a bread crumb back so people know to add to the list. I'd be glad to add that if you want.

Reply:
I did add a print